### ISSUE TITLE
Fail to find RTCOptions in 3.1.2 UNITY_EDITOR_OSX (3.1.1 works)

### ISSUE THREAD

**Issue by BestStream** [PlayEveryWare/eos_plugin_for_unity#620]
Library/PackageCache/com.playeveryware.eos@c13536130d/Runtime/Core/PlatformSpecifics.cs(91,55): error CS1061: 'Options' does not contain a definition for 'RTCOptions' and no accessible extension method 'RTCOptions' accepting a first argument of type 'Options' could be found (are you missing a using directive or an assembly reference?)

**Comment by arthur740212** [comment]
Hello, thanks for the report.
It looks like there is an extra unwanted line of code that was causing the problem.
Let's look at `MacOSPlatformSpecific.cs` line 35.
Remove the line 
```
   using Options = Epic.OnlineServices.IntegratedPlatform.Options;
```
That should solve your issue.

We will include this fix in the release shortly after.

**Comment by paulhazen** [comment]
I'm closing this issue as it has been resolved in PR #631. @BestStream please feel free to re-open with additional context if you are continuing to experience this issue after switching to the `3.1.3` release.
